grafana/README.md

```diff
@@ -8,14 +8,7 @@ The steps to access the customized dashboards are as follows:
 1. Login as admin/admin
 1. Change admin password
 1. Add a Prometheus datasource. Ensure the resource matches what you setup for pgmonitor, e.g. `localhost:9090`
-1. Download and import all dashboards to the datasource you created:
-  - [PostgreSQL.json](PostgreSQL.json)
```

Comment:
@keithf4 I understand why you removed this for making it concise, but the reason I had added it in is because I was confused when I initially went to download the data sources, so I wanted to make it explicit.

Perhaps we could bring it out as a table showing what the data sources are and what they do?

Reply:
It wasn't just being concise, it was because it would be something else that had to be maintained every time we added/removed a dashboard. Can possibly just be more clear in the description of where the dashboards are.

Reply:
@keithf4 How frequently are dashboards being added/removed? With that said, clarity works too 😄